src/input.rs

```diff
@@ -240,52 +239,43 @@ impl<'a, N: Notify + 'a> Processor<'a, N> {
                 32 + 1 + line.0 as u8,
             ];
 
-            self.ctx.notifier.notify(msg);
+            self.ctx.write_to_pty(msg);
         }
     }
 
     pub fn sgr_mouse_report(&mut self, button: u8, release: bool) {
-        let (line, column) = (self.ctx.mouse.line, self.ctx.mouse.column);
+        let (line, column) = (self.mouse.line, self.mouse.column);
         let c = if release { 'm' } else { 'M' };
 
         let msg = format!("\x1b[<{};{};{}{}", button, column + 1, line + 1, c);
```

Comment:
Maybe we should have some consts defined for ANSI escapes and `concat!(ANSI_CSI, format!(...)` so people can understand what this is supposed to do (including `m` and `M` in this context)